### PR TITLE
[TC2-1218] kazakhstan api doc

### PR DESCRIPTION
**Pull request by kalmerkaurson**

### CODE REVIEW

File: affiliates/kazakhstan.swagger.json [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
...ps://api.credy.eu/v3/)\n\nStaging URL: [http://api.staging.credy.eu/v3/](http://api.staging.credy.eu/v3/)\n\nAPI supports post fields, XML and JSON.\n\nTo get response error messages in Indonesian, \"Accept-Language\" header’s value must be set to according ISO code. Ex: `ka`.\n\n## Authorization\nAuthorization is performed with OAuth 2.0 Client Credentials Grant.\nWhen requesting token, the client key and secret key must be sent in request body.\nClient key and secret key will be assigned to you by Credy.\nThe requests to api must be authenticated with [bearer token](https://tools.ietf.org/html/rfc6749#section-7.1).\n\n## Lead registration flow\n\n1. Request is sent to /v3/customers\n\n2. If request is successful, customer uuid must be stored\n\n3. Request is sent to /v3/leads with previously stored customer uuid in query parameter \"customer\" *ex. /v3/leads?customer=a3ff7c20-e9e8-43a0-bfb7-71a5cf43b8d3* \n4. When the lead is accepted by api (under_investigation is received from POST lead), please redirect the customer to [https://www.credy.ge/loading/?lead={UUID}](https://www.credy.ge/loading/?lead={UUID}) (where {UUID} is replaced by the uuid that you receive from POST lead)\n"
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: credy.ge?

File: affiliates/kazakhstan.swagger.json [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+    "oAuth": []
+  }],
+  "paths" : {
+    "/customers" : {
+        "post" : {
+        "description" : "Register customer",
+        "parameters" : [ {
+        "name" : "Accept-Language",
+        "in" : "header",
+        "description" : "To get response error messages in kazakhstan, header’s value must be set to according ISO code. Ex: `kz` or `kz-KZ`.",
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: its ru-KZ

File: affiliates/kazakhstan.swagger.json [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+        "required" : false,
+        "style" : "simple",
+        "explode" : false,
+        "schema" : {
+          "format" : "string"
+        },
+        "example" : "kz-KZ"
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: its ru-KZ

File: affiliates/kazakhstan.swagger.json [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+        } ]
+      }
+    },
+    "/leads" : {
+      "post" : {
+        "description" : "Register lead",
+        "parameters" : [ {
+          "name" : "Accept-Language",
+          "in" : "header",
+          "description" : "To get response error messages in kazakhstan, header’s value must be set to according ISO code. Ex: `kz` or `kz-KZ`.",
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: its ru-KZ

File: affiliates/kazakhstan.swagger.json [hunk 5]
```diff
@@ -0,0 +1,466 @@
...
+          "required" : false,
+          "style" : "simple",
+          "explode" : false,
+          "schema" : {
+            "format" : "string"
+          },
+          "example" : "kz-KZ"
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: its ru-KZ

File: affiliates/kazakhstan.swagger.json [hunk 6]
```diff
@@ -0,0 +1,466 @@
...
+    },
+    "parameters" : {
+      "AcceptLanguageHeader" : {
+        "name" : "Accept-Language",
+        "in" : "header",
+        "schema" : {
+          "format" : "string"
+        },
+        "required" : false,
+        "description" : "To get response error messages in Indonesian, header’s value must be set to according ISO code. Ex: `ka`.",
```
Comment (Nuffic) [inline affiliates/kazakhstan.swagger.json]: indonesian?

**Review by Nuffic — CHANGES_REQUESTED** [review]

File: affiliates/kazakhstan.swagger.json (hunk 1 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

File: affiliates/kazakhstan.swagger.json (hunk 2 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

File: affiliates/kazakhstan.swagger.json (hunk 3 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

File: affiliates/kazakhstan.swagger.json (hunk 4 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

File: affiliates/kazakhstan.swagger.json (hunk 5 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

File: affiliates/kazakhstan.swagger.json (hunk 6 above)
Comment (kalmerkaurson) [reply affiliates/kazakhstan.swagger.json]: fixed

**Pull request merged**